app/commands.py

```diff
@@ -472,29 +467,30 @@ def associate_services_to_organizations():
     current_app.logger.info("finished associating services to organizations")
 
 
-@notify_command(name="populate-service-volume-intentions")
-@click.option(
-    "-f",
-    "--file_name",
-    required=True,
-    help="Pipe delimited file containing service_id, SMS, email",
-)
-def populate_service_volume_intentions(file_name):
-    # [0] service_id
-    # [1] SMS:: volume intentions for service
-    # [2] Email:: volume intentions for service
-
-    # TODO maintainability what is the purpose of this command? Who would use it and why?
-
-    with open(file_name, "r") as f:
-        for line in itertools.islice(f, 1, None):
-            columns = line.split(",")
-            current_app.logger.info(columns)
-            service = dao_fetch_service_by_id(columns[0])
-            service.volume_sms = columns[1]
-            service.volume_email = columns[2]
-            dao_update_service(service)
-    current_app.logger.info("populate-service-volume-intentions complete")
+# TODO maintainability what is the purpose of this command? Who would use it and why?
+# COMMENTING OUT UNTIL WE DETERMINE IF WE NEED IT OR NOT
+# @notify_command(name="populate-service-volume-intentions")
+# @click.option(
+#     "-f",
+#     "--file_name",
+#     required=True,
+#     help="Pipe delimited file containing service_id, SMS, email",
+# )
+# def populate_service_volume_intentions(file_name):
+#     # [0] service_id
+#     # [1] SMS:: volume intentions for service
+#     # [2] Email:: volume intentions for service
+
+
+#     with open(file_name, "r") as f:
+#         for line in itertools.islice(f, 1, None):
+#             columns = line.split(",")
+#             current_app.logger.info(columns)
+#             service = dao_fetch_service_by_id(columns[0])
+#             service.volume_sms = columns[1]
+#             service.volume_email = columns[2]
+#             dao_update_service(service)
+#     current_app.logger.info("populate-service-volume-intentions complete")
```

Comment:
We could look this up in the [UK Notify docs](https://www.notifications.service.gov.uk/using-notify), but I suspect if we haven't needed it by now, we could probably remove this from our version too.